scripts/run-analyses-for-figures.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#
+# Run all analysis modules that are used for manuscript figures
+
+#enviroment settings
+set -e
+set -o pipefail
+
+# If RUN_LOCAL is used, the time-intensive steps are skipped because they cannot
+# be run on a local computer -- the idea is that setting RUN_LOCAL=1 will allow for
+# local testing running/testing of all the other figures
+RUN_LOCAL=${RUN_LOCAL:-0}
+
+# Find current directory based on this script
+WORKDIR=$(dirname "${BASH_SOURCE[0]}")
+cd "$WORKDIR"
+
+# Get base directory of project
+cd ..
+BASEDIR="$(pwd)"
+cd -
+
+analyses_dir="$BASEDIR/analyses"
+
+#################################
+
+# Run modules that cannot be run locally due to memory requirements
+if [ "$RUN_LOCAL" -lt "1" ]; then
+  bash ${analyses_dir}/focal-cn-file-preparation/run-prepare-cn.sh       # Figures 2 and S3
+  bash ${analyses_dir}/snv-callers/run_caller_consensus_analysis-pbta.sh # Figure S2
+  bash ${analyses_dir}/snv-callers/run_caller_consensus_analysis-tcga.sh # Figure S2
+  bash ${analyses_dir}/copy_number_consensus_call/run_consensus_call.sh  # Figure S3 (heatmap)
+fi
+
+# Run the `oncoprint-landscape` module shell script, for Figure 2 and S3
+bash ${analyses_dir}/oncoprint-landscape/run-oncoprint.sh
+
+# Run the interaction plots script for Figures 3 and S3
+bash ${analyses_dir}/interaction-plots/01-create-interaction-plots.sh
+
+
+# Run the chromothripsis module, which requires the chromosomal-instability module, for Figure 3
+bash ${analyses_dir}/chromosomal-instability/run_breakpoint_analysis.sh
+bash ${analyses_dir}/chromothripsis/run-chromothripsis.sh
+
+# Run the tp53 classifier, for Figures 4 and S5
+#  Note this module is run earlier in this script than the relevant 
+#  figures' placement because these modules use the TP53 scores:
+#  `mutational-signatures` and `survival-analysis`
+OPENPBTA_TP53_FIGURES=1 bash ${analyses_dir}/tp53_nf1_score/run_classifier.sh
```

Comment:
I think the logic in this module is wrong? Or at least compared to the docs...

https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/2a4d691498c08cc985962447806ff5f2c2b92176/analyses/tp53_nf1_score/run_classifier.sh#L13-L15

And the logic:

https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/2a4d691498c08cc985962447806ff5f2c2b92176/analyses/tp53_nf1_score/run_classifier.sh#L74-L85

Should it be `if [[ "$RUN_FOR_FIGURES" -eq "1" ]]`  or am I confused?

Reply:
Coming back to this, I think the code is what we want, i.e., it is doing 

```
# If running for the purpose of figure generation (RUN_FOR_FIGURES will be 1), 
 # use the data in the analysis directory that has been freshly collapsed 
 ```

But the doc at the top of the script has things mixed up? 

At this point, if we move to an approach where everything _in this script_ is using files from the data release, we can go ahead and rip this out entirely?

Reply:
> Should it be if [[ "$RUN_FOR_FIGURES" -eq "1" ]] or am I confused?

You are not confused! Thanks for catching this. It should indeed be `eq "1"` here. Making that change now and updating that comment to appropriately match. 

> At this point, if we move to an approach where everything _in this script_ is using files from the data release, we can go ahead and rip this out entirely?

Yeah, I think we can do that. The `collapse-rnaseq` results don't change with subtyping, so I guess it shouldn't matter where we pull those files from?

Reply:
Wait read the second comment!

Reply:
Is this the second comment?
> If you use the data release for figures, you don't need this at all. It was intended to let you use results for figures, as that was an existing paradigm that we've since moved away from.

Meaning, we can rip out the whole flag.

Reply:
This is the second comment: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1543#discussion_r935936382

The docs were wrong, not the behavior of the code. If running for figures in the past, we did not want to use the data in the release. But yes, we can rip out the flag and just always use the data in the release.